components/Channel.php

```diff
@@ -91,7 +97,9 @@ public function getPropertyOptions($property)
 
     public function onRun()
     {
-        $this->addCss('assets/css/forum.css');
+        if ($this->property('includeStyles')) {
```

Comment:
@Web-VPF this should default to `true` if the includeStyles property is missing for some reason.
```suggestion
        if ($this->property('includeStyles', true)) {
```